metaseq/logging/progress_bar.py

```diff
@@ -388,3 +399,85 @@ def _log_to_wandb(self, stats, tag=None, step=None):
                 wandb.log({prefix + key: stats[key].val}, step=step)
             elif isinstance(stats[key], Number):
                 wandb.log({prefix + key: stats[key]}, step=step)
+
+
+try:
+    import functools
+
+    from aim import Repo as AimRepo
```

Comment:
Please add corresponding installation requirement to setup.py.

Reply:
sure, will push changes shortly

Reply:
done ✅